src/nationalarchives/components/nested-navigation/_index.scss

```diff
@@ -0,0 +1,157 @@
+.govuk-tna-tree {
```

Comment:
Is the name convention for the classes correct. At the moment in the frontend sass we've gone with this style: `.govuk-button__tna-button-link`, ie using double underscore. 

I seem to vaguely remember there being a reason for using that style.

Reply:
We are doing that sort of style though, there's things like `govuk-tna-tree__node-item__container` I'm not sure though, best to check with @garethfoote